public/client-v2/src/components/FrontsEdit/Fronts.js

```diff
@@ -26,15 +30,23 @@ class Fronts extends React.Component<FrontsComponentProps, ComponentState> {
     selectedFront: null
   };
 
+  componentWillMount() {
```

Comment:
These methods are being deprecated in future version of React.  For this one we can just set the state in the `constructor`. I actually don't think we should be using state here but actually just using `frontId` directly and using the prop to set the select value as it'll always be synced up no?